pkgs/development/python-modules/linuxpy/default.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pytestCheckHook,
+  python312Packages,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "linuxpy";
+  version = "0.20.0";
+
```

```suggestion
  pyproject = true;

```
You may need to add setuptools to build-system if the package needs it.